governance/votes/Executive vote - November 1, 2023.md

```diff
@@ -0,0 +1,92 @@
+date: 2023-11-01T00:00:00.000Z
+address: "$spell_address"
+
+
+---
+
+# [Executive Proposal] Spark Protocol-Aave Revenue Share Payment, DAO Resolution for HV Bank, Immunefi Security Core Unit MKR Vesting Transfer and Chainlog Housekeeping - November 1, 2023
+
+The Governance Facilitator teams, Sidestream, and Dewiz have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following actions within the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following actions will occur within the Maker Protocol:
+
+- The first Spark Protocol-Aave Revenue Share payment will be executed.
+- A DAO Resolution for the HV Bank vault (RWA009-A) will be approved, allowing the RWA Foundation to onboard with Galaxy Digital Trading Cayman LLC, an exchange agent.
+- A MKR vesting transfer will be executed for the Immunefi Security Core Unit, per its voluntary offboarding proposal.  
+- The GUSD and USDP Jar and SwapInputConduit contracts (associated with the GUSD-MakerDAO and Trident-MakerDAO partnerships) will be added to the Chainlog.
+
+**Voting for this executive proposal will place your MKR in support of the actions outlined above.**
+
+Unless otherwise noted, the actions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the actions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+
+---
+
+## Proposal Details
+
+### Spark Protocol-Aave Revenue Share Payment
+
+*Authorization*: [Support Scope Artifact](https://mips.makerdao.com/mips/details/MIP106#9-4-1-spark-protocol-aave-revenue-share)\
+*Proposal*: [Forum post](https://forum.makerdao.com/t/spark-aave-revenue-share-calculation-payment-1-q3-2023/22486) "Spark - Aave Revenue Share Calculation & Payment #1 [Q3 2023]"\
+*Authorizing Party:* [Support Scope Facilitator](https://forum.makerdao.com/t/spark-aave-revenue-share-calculation-payment-1-q3-2023/22486/2)
+
+The proposed action concerns executing the first payment of the Spark Protocol-Aave Revenue Share Agreement. If this executive proposal passes, Maker Governance will execute a payment from the Surplus Buffer of **2889 DAI** to a smart contract ([0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c](https://etherscan.io/address/0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c)) controlled by Aave Governance.
```

Comment:
From a technical standpoint, the DAI isn't directly sourced from the Surplus Buffer; instead, it's minted, and later, the equivalent amount is subtracted from the Surplus. To simplify the sentence:

If this executive proposal passes, Maker Governance will transfer **2889 DAI** to a smart contract ([0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c](https://etherscan.io/address/0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c)) controlled by Aave Governance.

Reply:
I'll leave this as is, as I wish to parallel the framing seen in the Forum post as well as the Support Scope.

Reply:
Alright, maybe it's a good idea to discuss this with the BA team and revise the Scopes, given that transfers cannot be made directly from the Surplus Buffer.